types/src/v0/impls/stake_table.rs

```diff
@@ -503,6 +511,27 @@ impl StakeTableState {
                 validator.stake_table_key = stake_table_key;
                 validator.state_ver_key = state_ver_key;
             },
+
+            StakeTableEvent::CommissionUpdate(CommissionUpdated {
```

Comment:
should add comments that we don't expect commission update events for protocol version V3 etc